service/internal/auth/authn.go

```diff
@@ -452,3 +456,17 @@ func validateDPoP(accessToken jwt.Token, acessTokenRaw string, dpopInfo dpopInfo
 	}
 	return &dpopKey, nil
 }
+
+func (a Authentication) isPublicRoute(path string) func(string) bool {
+	return func(route string) bool {
+		pattern := fmt.Sprintf("^%s$", route)
```

Comment:
Use `filepath.Match` here instead of regex

https://pkg.go.dev/path/filepath#Match

Reply:
I am not sure if filepath match is flexible enough here.  If I have these allowed routes `"/static/*", "/static/*/*"` the below test will fail. 

`s.Require().True(slices.ContainsFunc(s.auth.publicRoutes, s.auth.isPublicRoute("/static/test/next/maybe")))`

Reply:
Do we need it to?

Reply:
Also this is not a 'safe' way to convert to a regex. You should be splitting on the asterisks, then replacing each segment with a `regex.QuoteMeta` version of the segment

Reply:
Yea I like the glob approach better. I'm concerned that regex might introduce exposing a route that shouldn't be exposed unless we can bind it to the namespace.

Reply:
Its updated to use `filepath.Match for now`. We can circle back if/when we need a regex solution.